paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentSheetCardBrandFilter.kt

```diff
@@ -22,19 +22,11 @@ internal data class PaymentSheetCardBrandFilter(
 
             is PaymentSheet.CardBrandAcceptance.Allowed -> {
                 val isAllowed = brandCategory != null && cardBrandAcceptance.brands.contains(brandCategory)
```

Comment:
Could probably also inline these returns.

Reply:
Yeah agree, I kinda like how it reads though.